pilot/test/integration/testdata/mixer.yaml.tmpl

```diff
@@ -33,7 +33,7 @@ spec:
       serviceAccountName: istio-mixer-service-account
       containers:
       - name: mixer
-        image: {{.Hub}}/mixer:{{.Tag}}
+        image: {{.Hub}}/mixer_debug:{{.Tag}}
```

Comment:
there should be no difference between debug and non-debug go build of mixer, right?

Reply:
Yep, but this is the only make target I found. I didn't have it in me to investigate what is present in the non debug build of mixer. Atleast, we are now running with upto date mixer and istio-ca for all tests.

Reply:
so why do we have two build targets? it's a go binary, there's no difference in compiltion.